archinstall/lib/configuration.py

```diff
@@ -5,60 +5,72 @@
 	:type save:bool
 	"""
-	user_credentials = {}
-	disk_layout = {}
-	user_config = {}
-	for key in area:
-		if key in ['!users','!superusers','!encryption-password']:
-			user_credentials[key] = area[key]
-		elif key == 'disk_layouts':
-			disk_layout = area[key]
-		elif key in ['abort','install','config','creds','dry_run']:
-			pass
-		else:
-			user_config[key] = area[key]
+    user_credentials = {}
+    disk_layout = {}
+    user_config = {}
+    for key in area:
+        if key in ['!users', '!superusers', '!encryption-password']:
+            user_credentials[key] = area[key]
+        elif key == 'disk_layouts':
+            disk_layout = area[key]
+        elif key in ['abort', 'install', 'config', 'creds', 'dry_run']:
+            pass
+        else:
+            user_config[key] = area[key]
 
-	user_configuration_json = json.dumps({
-		'config_version': storage['__version__'], # Tells us what version was used to generate the config
-		**user_config, # __version__ will be overwritten by old version definition found in config
-		'version': storage['__version__']
-	} , indent=4, sort_keys=True, cls=JSON)
-	if disk_layout:
-		disk_layout_json = json.dumps(disk_layout, indent=4, sort_keys=True, cls=JSON)
-	if user_credentials:
-		user_credentials_json = json.dumps(user_credentials, indent=4, sort_keys=True, cls=UNSAFE_JSON)
+    user_configuration_json = json.dumps(
+        {
+            'config_version':
+                storage['__version__'],  # Tells us what version was used to generate the config
+            **user_config,  # __version__ will be overwritten by old version definition found in config
+            'version': storage['__version__']
+        },
+        indent=4,
+        sort_keys=True,
+        cls=JSON
+    )
+    if disk_layout:
+        disk_layout_json = json.dumps(disk_layout, indent=4, sort_keys=True, cls=JSON)
+    if user_credentials:
+        user_credentials_json = json.dumps(
+            user_credentials, indent=4, sort_keys=True, cls=UNSAFE_JSON
+        )
 
-	if save:
-		dest_path = pathlib.Path(storage.get('LOG_PATH','.'))
-		if (not dest_path.exists()) or not (dest_path.is_dir()):
-			log(f"Destination directory {dest_path.resolve()} does not exist or is not a directory,\n Configuration files can't be saved",fg="yellow",)
-			input("Press enter to continue")
-		else:
-			with (dest_path / "user_configuration.json").open('w') as config_file:
-				config_file.write(user_configuration_json)
-			if user_credentials:
-				target = dest_path / "user_credentials.json"
-				with target.open('w') as config_file:
-					config_file.write(user_credentials_json)
-			if disk_layout:
-				target = dest_path / "user_disk_layout.json"
-				with target.open('w') as config_file:
-					config_file.write(disk_layout_json)
+    if save:
+        dest_path = pathlib.Path(storage.get('LOG_PATH', '.'))
+        if (not dest_path.exists()) or not (dest_path.is_dir()):
+            log(
```

Comment:
This is silly to split into 3 lines because of word wrapping

Reply:
If you mean the arguments in the log function, I believe it tries to separate them into a new line for readability. 
There might be a setting for this, but it will most likely affect other places that will get less readable by it